tests/test_parser.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+
+from bankline_parser.data_services import parse_file, parse
+from bankline_parser.data_services.exceptions import ParseError
+
+
+class DataServicesParserTestCase(unittest.TestCase):
```

Comment:
For this, I wasn't thinking so much about the `is_valid` type checks so much as “has our code parsed the file correctly and given us the values we know are in the file”. Might be that's not necessary, given other tests, but might be nice to have it as an integration test type thing?